milestones/migrations/0001_initial.py

```diff
@@ -1,168 +1,147 @@
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
+                ('name', models.CharField(unique=True, max_length=25, db_index=True)),
+                ('description', models.TextField(blank=True)),
+                ('active', models.BooleanField(default=True)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='UserMilestone',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, verbose_name='created', editable=False)),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, verbose_name='modified', editable=False)),
+                ('user_id', models.IntegerField(db_index=True)),
+                ('source', models.TextField(blank=True)),
+                ('collected', models.DateTimeField(null=True, blank=True)),
+                ('active', models.BooleanField(default=True)),
+                ('milestone', models.ForeignKey(to='milestones.Milestone')),
+            ],
+        ),
+        migrations.AlterUniqueTogether(
+            name='milestone',
+            unique_together=set([('namespace', 'name')]),
+        ),
+        migrations.AddField(
+            model_name='coursemilestone',
+            name='milestone',
+            field=models.ForeignKey(to='milestones.Milestone'),
+        ),
+        migrations.AddField(
+            model_name='coursemilestone',
+            name='milestone_relationship_type',
+            field=models.ForeignKey(to='milestones.MilestoneRelationshipType'),
+        ),
+        migrations.AddField(
+            model_name='coursecontentmilestone',
+            name='milestone',
+            field=models.ForeignKey(to='milestones.Milestone'),
+        ),
+        migrations.AddField(
+            model_name='coursecontentmilestone',
+            name='milestone_relationship_type',
+            field=models.ForeignKey(to='milestones.MilestoneRelationshipType'),
+        ),
+        migrations.AlterUniqueTogether(
+            name='usermilestone',
+            unique_together=set([('user_id', 'milestone')]),
+        ),
+        migrations.AlterUniqueTogether(
+            name='coursemilestone',
+            unique_together=set([('course_id', 'milestone')]),
+        ),
+        migrations.AlterUniqueTogether(
+            name='coursecontentmilestone',
+            unique_together=set([('course_id', 'content_id', 'milestone')]),
+        ),
+        migrations.RunPython(forwards_func, reverse_func, False),
```

Comment:
`forwards_func, reverse_func, False`  False is for atomic, please check the link above.